paydb_endpoint.py

```diff
@@ -379,3 +435,4 @@ def transaction_info():
     if tx.user != api_key.user and tx.recipient != api_key.user:
         return bad_request(web_utils.UNAUTHORIZED)
     return jsonify(dict(tx=tx.to_json()))
+
```

Comment:
you added an extra new line which tripped up pylint

Reply:
deleted.